0008-quilt-ethos.md

```diff
@@ -0,0 +1,46 @@
+# Summary
+
+Quilt has a set of Community Guidelines and Governance RFCs. These provide a set of rules, expectations and guidance on what and how certain aspects of the project should be managed. After some reflection, it seems that a set of "Values" had not been define; the aim of this document is to set these values.
+
+# Motivation
+
+This can be seen as a derivate or inspiration from the Python Projects [PEP20](https://www.python.org/dev/peps/pep-0020/). I (ToffeeMax) have found this document useful when trying to both write better code using Python and as a general set of development principles. PEP20 could partially be seen as non-serious, however it is referenced within Python frequently. 
+
+The can be refered back to as more of a "Why" a decision was made - instead of the rigid proccess of HOW that other RFCs intend.
+
+
+# Explanation
+
+We will sometimes get things wrong, but:
+Do not let perfect be the enemy of actually shipping an idea.
+Modular, Simple and Flexible.
+Made by Anyone;
+Made for Anyone.
+This project breaths on collaboration.
+Not sure whether something will work? Try it. 
+Other will help shape it to something amazing.
```

Comment:
Probably meant "Others" here. It might be less "us vs. them" to say "Collaboration will shape it into something amazing" instead.